### ISSUE TITLE
Postgraphile V5 Support

### ISSUE THREAD

**Issue by ScottPierce** [graphile/postgis#58]
### Feature description

Postgraphile V5 Support

### Motivating example

I'd like to use this in postgrpahile v5

**Comment by benjie** [comment]
Is this something you're willing to take on, or would consider sponsoring?

**Comment by ScottPierce** [comment]
We already sponsor with pgrita, but we've considered giving more in the past.

**Comment by benjie** [comment]
Sorry, I should have been more clear; I was suggesting that you might sponsor development of this specific feature via a one-time donation, for example $2500.

(Also, though I appreciate you supporting Graphile Ltd by subscribing to our proprietary https://pgRITA.com service, that support is not sponsorship since it is paying for something that you cannot get without paying. Our sponsorship platforms are [GitHub Sponsors](https://github.com/sponsors/benjie) and [Patreon](https://patreon.com/benjie).)

**Comment by TimoStolz** [comment]
Maybe, some others could join and we sponsor together. I would have interest, too, and would be willing to sponsor €800.

@benjie, have you ever thought about crowdfunding for specific features? You have a strong community among your sponsors and, of course, the other users.

**Comment by benjie** [comment]
Yes, that's definitely something we've entertained. Polar.sh looks one of the best options right now, but I'm not sure how I feel about them having card fees _and_ taking at 10% cut on top of that. Might be worthwhile though - would allow people to put some pressure on me to prioritize specific issues (and also help to make the project more sustainable). I've another idea that I've been mooting with Jem for a while, but we've not had time to turn it into anything yet...

**Comment by dargmuesli** [comment]
I continued work on v5 compatibility in #66. Let me know if what I propose makes sense, I mainly based that work on the existing tests.
